### PR TITLE
Assignment 1 complete

### PR DESCRIPTION
**Pull request by jon7torres**

### CODE REVIEW

File: assignment1/assigment1 [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+            
+            if (part ==9){
+                numeral += "CM";
+            }
+            else if (part >=5){
+                numeral += 'D';
+                
+                for (int i =0; i < part; i++) {
+                    numeral += 'C';
+                }
```
Comment (benblazak) [inline assignment1/assigment1:46]: won't this output more `'C'`s than you want, for, say, `input == 700` ?

(looks like you got this right in the cases below)

File: assignment1/assigment1 [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+                
+            }
+            else if(part == 4){
+                numeral += "CD";
+            }
```
Comment (benblazak) [inline assignment1/assigment1]: i don't think `CD` is a valid sequence in roman numerals

File: assignment1/assigment1 [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    int input, part;
+    std::string numeral;
+    char yes;
+    
+    
+    do {
+        
+    std::cout << "Please enter an integer: ";
+    std::cin >> input;
+
```
Comment (benblazak) [inline assignment1/assigment1:25]: all this conversion stuff below should really be in its own function

**Comment by benblazak** [comment]
good, besides a few small comments.  however, regarding the submission: **please resubmit your pull request to the `assignment-01` repository to receive credit**.  it will not be considered late.  also, when you do that, please be sure to put a `.cpp` at the end of your filename, and to put the file in a directory as specified.  please review the video on google drive (see the link in the recent email) if needed.

after you submit a pull request to the correct repository, you may close it, unless you've made changes you wish me to review.

**Comment by james1997liu** [comment]
This is a continuation to my previous email about my grade, I was confused on what this email meant, I thought it was a conversation you had with another student so I ignored it, but would you mind further explain this email to me? Thanks so much

**Comment by benblazak** [comment]
Sounds like you figured it out, but if there's anything else, please let me know :-)

**Pull request closed without merging**